build/upstream-builds/build-kafkatool.sh

```diff
@@ -33,13 +33,19 @@ elif [[ ${MODE} == "UPSTREAM" ]]; then
 fi
 
 echo `pwd`
+CUR_DIR=${PWD}
 git clone https://github.com/airbnb/kafkat.git
 mkdir -p pnda-build
 mv kafkat kafka-tool-${KT_VERSION}
 cd kafka-tool-${KT_VERSION}
 git checkout tags/${KT_VERSION}
 gem build kafkat.gemspec
 mv kafkat*.gem kafkat-${KT_VERSION}.gem
+#Download all kafkat gem dependency gems
+GEM_LIST=$(<${CUR_DIR}/../../upstream-builds/dependencies/upstream-kafka-tool-dependencies.txt)
```

Comment:
replace CUR_DIR by PWD

Reply:
Removed CUR_DIR, not used PWD since it is not required.